sys/net/link_layer/ieee802154/submac.c

```diff
@@ -28,8 +28,8 @@
 #include "debug.h"
 
 #define CSMA_SENDER_BACKOFF_PERIOD_UNIT_US  (320U)
-#define ACK_TIMEOUT_US                      (864U)
-
+//#define ACK_TIMEOUT_US                      (864U)
+#define ACK_TIMEOUT_US                      (50000U)
```

Comment:
I already adapted the SubMAC in https://github.com/RIOT-OS/RIOT/pull/19198 to calculate the timeout on demand.
You would just need to define a channel page for LoRa and adapt the `ieee802154_get_symbol_time` function. For the LoRa channel page, I would try to define a very high value to avoid colliding with future 802.15.4 PHY specifications... let's say something in the range of 0xFF00 to 0xFFFF.